pkg/server/stream/v3/stream.go

```diff
@@ -23,20 +23,44 @@ type DeltaStream interface {
 
 // StreamState will keep track of resource state per type on a stream.
 type StreamState struct { // nolint:golint,revive
-	// Indicates whether the original DeltaRequest was a wildcard LDS/RDS request.
+	// Indicates whether the stream currently has a wildcard watch
```

Comment:
Let's make this comment explicit to delta since SOTW also uses this StreamState internally.